test/common/src/main/kotlin/org/codice/compliance/utils/TestCommon.kt

```diff
@@ -21,6 +21,7 @@ import org.apache.wss4j.common.util.DOM2Writer
 import org.codice.compliance.Common
 import org.codice.compliance.SAMLComplianceException
 import org.codice.compliance.saml.plugin.IdpResponse
+import org.codice.compliance.utils.decorators.IdpResponseDecorator
```

Comment:
❗️ Don't forget to format and optimize imports on all your changes. We do not currently have a build-time format checker running for Kotlin.